packages/@lwc/scoped-registry/src/create-scoped-registry.ts

```diff
@@ -24,12 +23,6 @@ export type CreateScopedConstructor = (
  * do not conflict with vanilla custom elements having the same tag name.
  */
 export function createScopedRegistry(): CreateScopedConstructor {
-    if (!hasCustomElements) {
-        // This code should never be reached, because we don't use the pivot registry if
-        // custom elements are unavailable.
-        throw new Error('Custom elements are not supported in this environment.');
-    }
-
```

Comment:
This check didn't seem worth keeping here. We're already checking `hasCustomElements` before we ever invoke `createScopedRegistry`, and extracting it into `@lwc/shared` seemed excessive.

If someone is using this library in an environment without custom elements, they're going to have a bad time... Although the browsers that don't support them are [vanishingly rare](https://caniuse.com/custom-elementsv1) now.